src/components/CustomDropdown.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div
+    class="dropdown"
+    :class="{ right: showRight, 'read-only': !method }"
+  >
+    <button
+      :class="{ active: showMenu }"
+      @click.prevent="showMenu = true"
+    >
+      <slot
+        name="diplayValue"
+      />
+      <span v-if="!$slots.diplayValue">{{ displayValue }}</span>
+      <img src="../assets/carretDown.svg">
+    </button>
+    <Modal
+      v-if="showMenu"
+      @close="showMenu = false"
+    >
+      <div
```

Comment:
I would like to read commits one by one, but I can't do so because it doesn't contain fixes that you have implemented later.

Reply:
Yes, the fixes are after the dropdown is renamed.